src/test/lib/optimizer/strategy/stored_table_column_alignment_rule_test.cpp

```diff
@@ -13,73 +14,97 @@ class StoredTableColumnAlignmentRuleTest : public StrategyBaseTest {
     Hyrise::get().storage_manager.add_table("t_a", load_table("resources/test_data/tbl/int_int_float.tbl", 1));
     Hyrise::get().storage_manager.add_table("t_b", load_table("resources/test_data/tbl/int_int_float.tbl", 1));
 
-    _stored_table_node_left = StoredTableNode::make("t_a");
-    _stored_table_node_right = StoredTableNode::make("t_a");
-
-    _stored_table_node_left->set_pruned_chunk_ids({ChunkID{2}});
-    _stored_table_node_right->set_pruned_chunk_ids({ChunkID{2}});
-
-    _stored_table_node_left->set_pruned_column_ids({ColumnID{0}});
-    _stored_table_node_right->set_pruned_column_ids({ColumnID{0}});
+    _stn_left = StoredTableNode::make("t_a");
+    _stn_left->set_pruned_chunk_ids({ChunkID{2}});
+    _stn_left->set_pruned_column_ids({ColumnID{0}});
+    _stn_right = std::static_pointer_cast<StoredTableNode>(_stn_left->deep_copy());
 
     _union_node = UnionNode::make(SetOperationMode::All);
-    _union_node->set_left_input(_stored_table_node_left);
-    _union_node->set_right_input(_stored_table_node_right);
+    _union_node->set_left_input(_stn_left);
+    _union_node->set_right_input(_stn_right);
 
     _rule = std::make_shared<StoredTableColumnAlignmentRule>();
   }
 
   std::shared_ptr<StoredTableColumnAlignmentRule> _rule;
   std::shared_ptr<UnionNode> _union_node;
-  std::shared_ptr<StoredTableNode> _stored_table_node_left, _stored_table_node_right;
+  std::shared_ptr<StoredTableNode> _stn_left, _stn_right;
```

Comment:
Why? :(

Reply:
Shorter, less bulky. No issue though, I revert it.